code/__DEFINES/mob_defines.dm

```diff
@@ -266,7 +266,8 @@
 #define isAutoAnnouncer(A)	(istype((A), /mob/living/automatedannouncer))
 
 #define iscameramob(A)	(istype((A), /mob/camera))
-#define isAIEye(A)		(istype((A), /mob/camera/aiEye))
+#define isAIEye(A)		(istype((A), /mob/camera/eye))
+#define isholo(A)			(istype((A), /mob/camera/eye/hologram))
```

Comment:
```suggestion
#define isholo(A)			(istype((A), /mob/camera/eye/hologram))
```
probably best to rename this to ishologram or something due to holoparas also using the holo word